specification/network/resource-manager/Microsoft.Network/stable/2018-12-01/webapplicationfirewall.json

```diff
@@ -0,0 +1,575 @@
+          }
+        }
+      }
+    },
+    "CustomRules": {
+      "description": "Defines contents of custom rules",
+      "properties": {
+        "rules": {
+          "description": "List of rules",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/CustomRule"
+          }
+        }
+      }
+    },
+    "CustomRule": {
+      "description": "Defines contents of a web application rule",
+      "required": [  
+        "priority",  
+        "ruleType",  
+        "matchConditions",  
+        "action"  
+      ], 	  
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Gets name of the resource that is unique within a policy. This name can be used to access the resource.",
+          "maxLength": 128
+        },
+        "etag": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Gets a unique read-only string that changes whenever the resource is updated."
+        },        
+        "priority": {
+          "description": "Describes priority of the rule. Rules with a lower value will be evaluated before rules with a higher value",
+          "type": "integer"
+        },        
+        "ruleType": {
+          "description": "Describes type of rule",
+          "type": "string",
+          "enum": [
+            "MatchRule"
+          ],
+          "x-ms-enum": {
+            "name": "ruleType",
+            "modelAsString": true
+          }
+        },
+        "matchConditions": {
+          "description": "List of match conditions",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/MatchCondition"
+          }
+        },
+        "action": {
+          "description": "Type of Actions",
+          "type": "string",
+          "enum": [
+            "Allow",
+            "Block",
+            "Log"
+          ],
+          "x-ms-enum": {
+            "name": "action",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "Transform": {
+      "description": "Describes what transforms applied before matching",
+      "type": "string",
+      "enum": [
+        "Lowercase",
+        "Trim",
+        "UrlDecode",
+        "UrlEncode",
+        "RemoveNulls",
+        "HtmlEntityDecode"
+      ],
+      "x-ms-enum": {
+        "name": "transform",
+        "modelAsString": true
+      }
+    },
+    "MatchVariable": {
+      "description": "Define match variables",
+      "required": [  
+        "name"
+      ],
+      "properties": {
+        "name": {
+          "description": "Match Variable",
+          "type": "string",
+          "enum": [
+            "RemoteAddr",
+            "RequestMethod",
+            "QueryString",
+            "PostArgs",
+            "RequestUri",
+            "RequestHeaders",
+            "RequestBody",
+            "RequestCookies"
+          ],
+          "x-ms-enum": {
+            "name": "transform",
```

Comment:
This is a duplicate of https://github.com/Azure/azure-rest-api-specs/pull/5185/files#diff-44386327d1e83b27a591055afce37c92R453 with different enum values.

Reply:
Thanks for catching this. Renamed the enum.